synapse/models/science.py

```diff
@@ -0,0 +1,92 @@
+
+                ('sci:evidence', ('guid', {}), {
+                    'doc': 'An assessment of how an observation supports or refutes a hypothesis.'}),
+            ),
+
+            'edges': (
+                (('sci:experiment', 'uses', None), {
+                    'doc': 'The experiement used the target nodes when it was run.'}),
+                (('sci:observation', 'has', None), {
+                    'doc': 'The observations are summarized from the target nodes.'}),
+                (('sci:evidence', 'has', None), {
+                    'doc': 'The evidence includes observations from the target nodes.'}),
+            ),
+
+            'forms': (
+                # TODO many of these forms need author/contact props
+                ('sci:hypothesis:type:taxonomy', {}, {}),
+                ('sci:hypothesis', {}, (
+
+                    ('name': ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'The name of the hypothesis.'}),
+
+                    ('type', ('sci:hypothesis:type:taxonomy', {}), {
+                        'doc': 'A taxonomy of hypothesis types.'}),
+
+                    ('summary': ('str', {}), {
+                        'doc': 'A summary of the hypothesis.'}),
+                )),
+
+                # TODO eventually link to a procedure form
+                ('sci:experiment', {}, (
+
+                    ('name', ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'The name of the experiment.'}),
+
+                    ('summary', ('str', {}), {
+                        'doc': 'A summary of the experiment.'}),
+
+                    ('type', ('sci:experiment:type:taxonomy', {}), {
+                        'doc': 'A taxonomy of experiment types.'}),
+
+                    ('window', ('ival', {}), {
+                        'doc': 'The time window where the experiment was run.'}),
+
+                )),
+
+                ('sci:observation', {}, (
+
+                    ('experiment', ('sci:experiment', {}), {
+                        'doc': 'The experiment which produced the observation.'}),
+
+                    ('summary', ('str', {}), {
+                        'doc': 'A summary of the observation.'}),
+
+                    ('time', ('time', {}), {
+                        'doc': 'The time that the observation occurred.'}),
+                )),
+
+                ('sci:evidence', {}, (
+
+                    ('hypothesis', ('sci:experiment', {}), {
+                        'doc': 'The hypothesis which which the evidence supports or refutes.'}),
+
+                    ('observation', ('sci:observation', {}), {
+                        'doc': 'The observation which supports or refutes the hypothesis.'}),
+
+                    ('summary', ('str', {}), {
+                        'doc': 'A summary of how the observation supports or refutes the hypothesis.'}),
+
+                    ('refutes', ('bool', {}), {
```

Comment:
I assume there was a discussion but I'm curious why we'd use `refutes` vs. `supports`?